docs/cybersecurity/ddos/index.md

```diff
@@ -0,0 +1,217 @@
+---
+seo:
+  title: Detecting a Slowloris DDoS attack 
+  description: A ksqlDB recipe that detects network disruption attacks by processing packet data
+---
+
+# Detect a Slowloris DDoS network attack 
+
+Distributed denial-of-service (DDoS) attacks are specific type of cyber-attack in which a targeted system is flooded with spurious network requests using multiple hosts and IP addresses. The distributed nature of these attacks make them more effective and difficult to mitigate. This recipe shows a strategy for ingesting and processing network packet data in an effort to detect a specific DDoS attack known as Slowloris.
+
+![slowloris_attack](../../img/ssh-attack.jpg)
+
+## Step by step
+
+### Set up your environment
+
+Provision a Kafka cluster in [Confluent Cloud](https://www.confluent.io/confluent-cloud/tryfree/?utm_source=github&utm_medium=ksqldb_recipes&utm_campaign=slowloris_ddos).
+
+--8<-- "docs/shared/ccloud_setup.md"
+
+### Read the data in
+
+--8<-- "docs/shared/connect.md"
+
+This recipe assumes you have captured your network packet data and published it to a RabbitMQ queue in JSON format. An example packet may look like the following example. 
+
+**Note**: To keep this recipe brief, some fields have been removed and names simplified from a typical packet capture event.
+
+```json
+{
+  "timestamp": "1590682723239",
+  "layers": {
+    "frame": { 
+      "time": "May 28, 2020 11:48:43.239564000 CST",
+      "protocols": "eth:ethertype:ip:tcp"
+    },
+    "eth": { 
+      "src": "FF:AA:C9:83:C0:21",
+      "dst": "DF:ED:E3:91:D4:13"
+    },
+    "ip": {
+      "src": "192.168.33.11",
+      "src_host": "192.168.33.11",
+      "dst": "192.168.33.77",
+      "dst_host": "192.168.33.77"
+    },
+    "tcp": { 
+      "srcport": "59202",
+      "dstport": "443",
+      "flags_ack": "1",
+      "flags_reset": "0"
+    }
+  }
+}
+```
+
+This connector will source that data into a Kafka topic and can be stream processed in ksqlDB.
+
+```json
+--8<-- "docs/cybersecurity/ddos/source.json"
+```
+
+--8<-- "docs/shared/manual_insert.md"
+
+### ksqlDB code
+
+This application takes the raw network packet data and creates a structured stream of events that can be processed using SQL. Using [Windows](https://docs.ksqldb.io/en/latest/concepts/time-and-windows-in-ksqldb-queries/#windows-in-sql-queries) and filters, the application detects a high number of connection `RESET` events from the server and isolates the potentially offending source.
```

Comment:
If `Windows` is linked, maybe `filters` should be as well?  Or neither.

Reply:
I couldn't find a good single link to "filters" or conditionals, but I may have missed it. I'm ok with the minor inconsistency, but if you want me to remove the windows link I can